misc/loop/scripts/switch.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env sh
+
+set -e
+
+# This script goal is to:
+# 1. Snapshot txs for current running node
+# 2. Start a new node with this tx in genesis file
+# 3. switch traefik node
+
+
+BACKUP_DIR="/backups"
+
+NOW=$(date +'%Y-%m-%d_%s')
+BACKUP_FILE="${BACKUP_DIR}/backup_${NOW}.jsonl"
+BACKUP_LEGACY_FILE=$(echo ${BACKUP_FILE} | sed 's/.jsonl$/-legacy.jsonl/')
+
+HOST_PWD="${HOST_PWD:=$(pwd)}"
+
+TX_ARCHIVE_CMD=${TX_ARCHIVE_CMD:-"tx-archive"}
+
+CONTAINER_NAME="gno-${NOW}"
+
+# Set the current portal loop in READ-ONLY mode
+sed -i -E 's/middlewares: \[.*\]/middlewares: []/' /etc/traefik/configs/gno.yml
+
+# Get latest version of gno
+docker pull ghcr.io/gnolang/gno
```

Comment:
```suggestion
# Get latest version of gno
docker pull ghcr.io/gnolang/gno

# Set the current portal loop in READ-ONLY mode
sed -i -E 's/middlewares: \[.*\]/middlewares: []/' /etc/traefik/configs/gno.yml
```

Reply:
Why this decision ? I was thinking to add a system if docker pull don't get a new version, to not update the portal loop.

Reply:
Yes, you should stop the script if the `docker pull` fails by using `|| exit 1` or a similar method.

My modification prevents unnecessary activation of read-only (maintenance) mode and reduces the duration of maintenance mode.

The objective is to minimize the number of non-instant commands executed after entering maintenance mode and to switch to maintenance mode only when we really need it, i.e., when we have two running instances simultaneously.